packages/pluggableWidgets/accordion-native/src/ui/Styles.ts

```diff
@@ -0,0 +1,87 @@
+import { Style } from "@mendix/piw-native-utils-internal";
+import { ViewStyle, TextStyle } from "react-native";
+
+export interface AccordionStyle extends Style {
+    container: ViewStyle;
+    group: AccordionGroupStyle;
+}
+
+export interface AccordionGroupStyle {
+    container: ViewStyle;
+    header: {
+        container: ViewStyle;
+        heading1: TextStyle;
+        heading2: TextStyle;
+        heading3: TextStyle;
+        heading4: TextStyle;
+        heading5: TextStyle;
+        heading6: TextStyle;
+        icon: AccordionIconStyle;
+    };
+    content: ViewStyle;
+}
+
+export interface AccordionIconStyle extends ViewStyle {
+    size: number;
+    color: string;
+}
+
+export const defaultAccordionStyle: AccordionStyle = {
+    container: {
+        backgroundColor: "#FFF"
```

Comment:
Does the user need to configure the widget with some class to get it to render full width?